src/components/CategoryFilter.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+
+function CategoryFilter(props) {
+  const CATEGORIES = ["ALL", "Action", "Biography", "History", "Horror", "Kids", "Learning", "Sci-Fi"]
+
+  return (
+    <select name="category" onChange={(e) => props.onChange(e)}>
+    <option value='default'>Filter Books</option>
```

Comment:
Hey, this is not needed, i should be seeing `All` its all the books that are displayed at start after all.